article/app/views/fragments/articleBody.scala.html

```diff
@@ -15,7 +15,11 @@
             <meta itemprop="mainEntityOfPage" content="@LinkTo(article.url)">
             <div itemprop="publisher" itemscope itemtype="https://schema.org/Organization">
                 <meta itemprop="name" content="The Guardian">
-                <link itemprop="sameAs" content="http://www.theguardian.com">
+                @if(amp) {
+                    <meta itemprop="logo" content="/assets/images/favicons/152x152.png">
```

Comment:
is it worth putting a comment just to say they can't handle sameAs yet so we just need to duplicate the logo?